src/api/getBaladiyaByCodeAndDairaCodeAndWilayaCode.js

```diff
@@ -0,0 +1,31 @@
+const projectBaladiya = require('../utils/projections/projectObject');
+
+const getBaladiyaByCodeAndDairaCodeAndWilayaCode = (data) =>
```

Comment:
Please do you have any use case for such a feature? I believe that Baladyia code is unique overall, so if you'd use it it won't matter to add extra params. And if you want to get a baladyia by wilaya or daira code you'd just use their methods with a projection

Reply:
can you explain how can i get the same result by using a projection

Reply:
We do have these methods in place:

```js
const {
  getBaladyiatsForWilaya,
  getBaladyiatsForDaira
} = require("@dzcode-io/leblad");

const rs1 = getBaladyiatsForWilaya(31); // if this wasn't implemented, we'd use getWilayaByCode and project only "dairats"

const rs2 = getBaladyiatsForDaira("ORAN");
```

Maybe it'd be nice to have another `getBaladyiatsForDairaCode` method.